launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/Launcher.kt

```diff
@@ -99,7 +101,7 @@ class Launcher(val args: Array<String>) {
     fun runBuildWrapperOnce() = workFile("gradle/wrapper/gradle-wrapper.properties") {
         if (!exists()) {
             println("Generating Gradle wrapper files")
-            runBuild(listOf("wrapper", "-Plauncher.wrapper=true"))
+            runBuild(listOf("wrapper", "-Plauncher.wrapper=true", *wrapperArgs.toTypedArray()))
```

Comment:
Not sure whether there is a better way to do this. I don’t code in kotlin that often.
Maybe switch the `runBuild` method to varargs?

Reply:
>  Maybe switch the runBuild method to varargs?

I tried that and it requires adding the `*….toTypedArray()` construct in two other places so there’s nothing to be gained from that.

Reply:
```suggestion
            runBuild(listOf(listOf("wrapper", "-Plauncher.wrapper=true"), wrapperArgs).flatten())
```
would be an option

Reply:
I switched to the nested list variant